bridge-stp.in

```diff
@@ -80,20 +81,29 @@ MSTPD_ARGS=''
 # A space-separated list of bridges for which MSTP should be used in place of
 # the kernel's STP implementation.  If empty, MSTP will be used for all bridges.
 MSTP_BRIDGES=''
+LOGGER='logger -t bridge-stp -s'
```

Comment:
Use --tag and --stderr so others can see what this is doing without having to check the man page for `logger`.